launcher/sdw-launcher.py

```diff
@@ -3,12 +3,33 @@
 from sdw_updater_gui.UpdaterApp import UpdaterApp
 from sdw_util import Util
 from sdw_updater_gui import Updater
-
+from sdw_updater_gui.UpdaterApp import launch_securedrop_client
+from sdw_updater_gui.Updater import should_launch_updater
 import logging
 import sys
+import argparse
+
+DEFAULT_INTERVAL = 28800
```

Comment:
I always make a habit of annotating seconds values, i.e. `# 8 hours` in this case

Reply:
:heavy_check_mark: